src/components/generic/Banner/Banner.scss

```diff
@@ -0,0 +1,12 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-banner {
+  background: var(--wxc-banner-background);
+  height: 120px;
```

Comment:
the height should be auto, put some paddings to aquire the total height in figma